src/app/modules/browse/browse-controller.js

```diff
@@ -83,23 +83,38 @@
                 params.bookOrBooks = _localize('book');
             }
 
+            var isFiltered = false;
+
             if (count === 0) {
-                message = _localize('There are no books that match your search for {shelf} {language} books', params);
+                $scope.resultsMessage = _localize('There are no books that match your search', params);
```

Comment:
Can you say why you did this? At first glance, I think it's better how it was. Localizers just need to include the bracketed material somewhere in their translation. They're used to having to do that with  {0}'s and {1}'s in .net-land, here it's nice because they get actual labels.